packages/turf-line-split/test.js

```diff
@@ -20,18 +21,31 @@ const fixtures = fs.readdirSync(directories.in).map(filename => {
     };
 });
 
-test('turf-line-split', t => {
-    for (const {filename, name, geojson}  of fixtures) {
-        const line = geojson.features[0];
-        const splitter = geojson.features[1];
-        const results = colorize(lineSplit(line, splitter));
-        featureEach(geojson, feature => results.features.push(feature));
-
-        if (process.env.REGEN) write.sync(directories.out + filename, results);
-        t.deepEquals(results, load.sync(directories.out + filename), name);
-    }
+// test('turf-line-split', t => {
+//     for (const {filename, name, geojson}  of fixtures) {
+//         const line = geojson.features[0];
+//         const splitter = geojson.features[1];
+//         const results = colorize(lineSplit(line, splitter));
+//         featureEach(geojson, feature => results.features.push(feature));
+
+//         if (process.env.REGEN) write.sync(directories.out + filename, results);
+//         t.deepEquals(results, load.sync(directories.out + filename), name);
+//     }
+//     t.end();
+// });
+
+test('turf-line-split -- lines should split the same feature 1 with 2 as 2 with 1', t => {
+    const featureOne = lineString([[114.58215786065353825, -14.82470576519326144], [137.21678649707752129, -16.71692980416107588]]);
+    const featureTwo = lineString([[119.1412061636556956, -19.83670052919270788], [133.06640625, -12.64033830684678961]]);
+
+    const resultsOne = lineSplit(featureOne, featureTwo);
+    const resultsTwo = lineSplit(featureTwo, featureOne);
+
+    const midCoordOne = getCoords(resultsOne.features[0])[1];
+    const midCoordTwo = getCoords(resultsTwo.features[1])[0];
+    t.deepEquals(midCoordOne, midCoordTwo, 'Splits were made in different locations')
```

Comment:
Bit better, however kinda hard to troubleshoot if there isn't a GeoJSON output... typically we save all our outputs in `test/out` that way we can easily preview the expected or NOT expected GeoJSON outputs.